maven-java/kalix-maven-archetype-spring-value-entity/README.md

```diff
@@ -0,0 +1,27 @@
+# kalix-maven-archetype
+
+This archetype can be used to generate a project suitable for the development of a Service using a Value Entity with [Kalix](https://docs.kalix.io).
+
+**The Maven project created from the archetype will contain a Spring Value entity.**
+
+For the latest release see [GitHub releases](https://github.com/lightbend/kalix-jvm-sdk/releases).
+
+To kickstart a project on Linux and macOS:
+
+```shell
+mvn \
+  archetype:generate \
+  -DarchetypeGroupId=io.kalix \
+  -DarchetypeArtifactId=kalix-maven-archetype-spring-value-entity \
```

Comment:
The value entity archetype we have doesn't contain `-value-entity`. 

The goal is to have a short name for our default archetype. It happens to be one have has a value entity already encoded.

Reply:
After our discussion @octonato, deciding to clean the content of the archetype the comment above is no longer relevant. I'll use a short name as you suggested.